tests/credentials_test.go

```diff
@@ -38,6 +38,22 @@ import (
 	cd "kubevirt.io/kubevirt/tests/containerdisk"
 )
 
+func vmiLauncher(virtClient kubecli.KubevirtClient) func(vmi *v1.VirtualMachineInstance) *v1.VirtualMachineInstance {
```

Comment:
is this being used outside of the credentials_test? if so lets move it to some shared package

Reply:
I use it in the probes_test as well now, but not sure if I can get rid of it there.
Could be worth moving.

Reply:
lets move it to tests/utils.go at least since it is shared.